content/post/2018-12-10-heart.md

```diff
@@ -0,0 +1,185 @@
+---
+title: You Gotta Have Heart
+date: 2018-12-10
+author: Doug Selph
+draft: false
+tags: ["nerves", "heart", "watchdog"]
+---
+
+It’s one of an embedded developer’s worst nightmares: You have devices in the
+field, on remote sites, where you don’t have access to local staff to serve as
+your hands. You’re far from the devices, maybe on a different continent, even,
+and now you have a device that’s no longer responding to network requests, or
+not sending event data as expected.
+
+It is a terrible feeling.
+
+If you’re using Nerves to build firmware, you have tools at your disposal to
+help keep your devices online.
+
+## Use Shoehorn
+
+First things first, if you’re deploying Nerves firmwares, make sure you’re
+using Shoehorn in the most advantageous way. `Nerves.Shoehorn`  allows you to
+have a list of applications started on the BEAM VM prior to your application
+being started.
+
+You want to make certain that you get `Nerves.Network` started from the
```

Comment:
You should probably refer to `Nerves.Network` and `Nerves.Firmware.SSH` by their atom application names for consistency with the config example below.